_includes/defined-schema/indexes.md

```diff
@@ -0,0 +1,26 @@
+# Indexes
+
+To optimize the Parse Server performance you can define indexes and compound indexes.
+
+To define an index on a `Pointer` field you need to use a
```

Comment:
Reformat and reword to:

To optimize the Parse Server performance you can define indexes and compound indexes. Parse Server does not support indexes on special `_Join` classes used under the hood by the `Relation` type.

To define an index on a `Pointer` field you need to use the special notation `_p_<FIELDNAME>`. For example, if you define `city: { type: "Pointer", targetClass: "City" }` in your `fields` you can define an index on this pointer with `cityIndexExample: { _p_city: true }`.